### DISCUSSION TITLE
Installation help/Running simulation

### DISCUSSION THREAD

**Discussion by childc3** [SpaceTeam/ortools#15] [General]
Hi, would you be able to provide assistance with running a simulation? When trying to use my own ork file I get a "MotorIgnitionException" message, and if I try modifying the example ork file with a different motor I get an "IndexError: too many indices for array." I'm not sure if I installed and set up everything properly.

**Comment by christophfroehlich** [comment]
Hi! Can you run the [examples](https://github.com/SpaceTeam/ortools#usage) from this repository? Do you have an ork file for the correct openrocket version (15.03)? Could you upload it here?

**Reply by childc3** [reply]
Thanks for the quick response, I can run the simple.ork example just fine, but for the threestage.ork it runs the first simulation, then has the error `WARNING:root:Simulation of stage 1 threw an exception
WARNING:root:Simulation of stage 2 threw an exception`
Here's the ork file that I'm trying to get work:
[TwoStager.zip](https://github.com/SpaceTeam/ortools/files/13340325/TwoStager.zip)

**Reply by childc3** [reply]
I don't know if this helps but I'm running Java 8 update 391.
